### ISSUE TITLE
ADD_VPOOL failed: mkdir: cannot create directory /mnt/geo-accel-alba: File exists'

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#968]
During a extend of a vpool we saw this:
Exception: `UnicodeDecodeError('ascii', 'mkdir: cannot create directory \xe2\x80\x98/mnt/geo-accel-alba\xe2\x80\x99: File exists', 31, 32, 'ordinal not in range(128)')`

Extended exception: 

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 700, in add_vpool
    root_client.dir_create(dirs2create)
  File "/opt/OpenvStorage/ovs/extensions/generic/sshclient.py", line 275, in dir_create
    self.run('mkdir -p "{0}"; echo true'.format(directory))
  File "/opt/OpenvStorage/ovs/extensions/generic/sshclient.py", line 59, in inner_function
    return outer_function(self, *args, **kwargs)
  File "/opt/OpenvStorage/ovs/extensions/generic/sshclient.py", line 250, in run
    error = self._clean_text(stderr.readlines())
  File "/opt/OpenvStorage/ovs/extensions/generic/sshclient.py", line 207, in _clean_text
    return text.strip().replace(u'\u2018', u'"').replace(u'\u2019', u'"')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 31: ordinal not in range(128)
```

**Comment by kinvaris** [comment]
Looking further it seemed like the mountpoint was not unmounted from fuse.

**Comment by khenderick** [comment]
Todo:
- Check for free mountpoint `/mnt/<vpool name>` in wizard + API prechecks
- Figure out this new shitty unicode character

**Comment by wimpers** [comment]
Partly duplicates https://github.com/openvstorage/framework/issues/626 ?

**Comment by khenderick** [comment]
Yes, this only has this unicode thingy as extra

**Comment by khenderick** [comment]
Fixed by #1038, packaged in openvstorage-2.7.4-rev.4147.6f02d53

**Comment by JeffreyDevloo** [comment]
### Information
This ticket is a summary of two other issues:
- https://github.com/openvstorage/framework/issues/1039
- https://github.com/openvstorage/framework/issues/626

Both of these are closed, so closing this one
